scripts/test_e2e.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -ex
+
+export BROWSERSTACK_USERNAME=andrewwylde1
+
+rm -rf .dfx
+npm install
+
+dfx start --background
+dfx canister create --all
+dfx build
+dfx canister install --all
+
+CAN_JSON="{\"linkedup_assets\": \"$(dfx canister id linkedup_assets)\"}"
+
+# run e2e
+echo $CAN_JSON >cypress/plugins/canister_ids.json
+BrowserStackLocal --key $BROWSERSTACK_APIKEY --daemon start
```

Comment:
If you may want something earlier in the script to complain if `BROWSERSTACK_APIKEY` is not set

```
if [ -z "$BROWSERSTACK_API_KEY" ]; then
  2>&1 echo "BROWSERSTACK_API_KEY environment variable is required"
  #exit 1
fi
```